### ISSUE TITLE
[shelly] IllegalArgumentException when switching Shelly Vintage on

### ISSUE THREAD

**Issue by markus7017** [openhab/openhab-addons#9395]
Sending ON to the brightness channel of a Shelly Vintage bulb results in
```
Unable to update channel white#brightness$Value with -1 % (type class org.eclipse.smarthome.core.library.types.QuantityType): Value must be between 0 and 100 (class java.lang.IllegalArgumentException)
```

**Comment by markus7017** [comment]
will be fixed with PR #9354

**Comment by markus7017** [comment]
will be fixed with PR #9508
